packages/victory-area/src/index.d.ts

```diff
@@ -27,3 +28,14 @@ export interface VictoryAreaProps
  * Add VictoryArea as a child of VictoryChart for a complete chart.
  */
 export class VictoryArea extends React.Component<VictoryAreaProps, any> {}
+
+export interface VictoryAreaPrimitiveProps extends VictoryCommonPrimitiveProps {
```

Comment:
This is a nitpick, but I feel like the naming for the primitive props interface should match the naming of the component, so `AreaProps` instead of `VictoryAreaPrimitiveProps`.

Reply:
That's fair. The primitive interfaces exported from victory-core all begin with "Victory" (e.g. VictoryBorder), but obviously for something like Area there's going to be collision if it's called "VictoryAreaProps", so I was trying to find some sort off middle ground for consistency there, but I don't really like it either.

Is it ok if I rename exported primitive/simple interfaces in the victory-core defs by removing "Victory", too?

Reply:
Yes, please do! That sounds like the most consistent thing.